### PR TITLE
Use HTTPS in Management CLI 

### LINKED ISSUE #176: Admin services API should use HTTPs

### PR DESCRIPTION
**Pull request by SanojPunchihewa**
## Purpose
Resolves #176 

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: cmd/utils/utils.go [hunk 1]
```diff
@@ -34,6 +35,7 @@ import (
 // Invoke http-post request using go-resty
 func InvokePOSTRequest(url string, headers map[string]string, body string) (*resty.Response, error) {
 
+    resty.SetTLSClientConfig(&tls.Config{ InsecureSkipVerify: true })
```
Comment (a5anka) [inline cmd/utils/utils.go]: Can we extract a method for this and call it in line 47 as well to avoid code duplication?

File: cmd/README.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ Command Line tool for managing the WSO2 Micro Integrator
 - ### Management API Address and Port
     To configure the address and the port of the Management Api in the CLI use the [**init**](#init) command. This will generate a file called server_config.yaml which contains the address and the port. If the init command was not used, the address and the port will have the default values
 
-    NOTE: The default address is http://localhost and the port is 9201
+    NOTE: The default address is https://localhost and the port is 9154
```
Comment (Nirothipan) [inline cmd/README.md]: This has to be 9154 + 10 ( offset )

File: cmd/README.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ Command Line tool for managing the WSO2 Micro Integrator
             mi init
             Enter following parameters to configure the cli
             Host name(default localhost): abc.com
-            Port number(default 9201): 9595
+            Port number(default 9154): 9595
```
Comment (Nirothipan) [inline cmd/README.md]: 9164

File: cmd/cmd/init.go [hunk 4]
```diff
@@ -32,35 +32,56 @@ const initCmdShortDesc = "Set Management API configuration"
...
+    fmt.Print("Host name(default localhost): ")
     var host string
     fmt.Scanln(&host)
     if len(host) == 0 {
-        fmt.Println("Host name is not specified, default value is used")
+        fmt.Println("Host name is not specified, default value localhost is used")
         host = utils.DefaultHost
     }
-    fmt.Print("Enter Port number: ")
+    fmt.Print("Port number(default 9154): ")
```
Comment (Nirothipan) [inline cmd/cmd/init.go]: 9164

File: cmd/cmd/init.go [hunk 5]
```diff
@@ -32,35 +32,56 @@ const initCmdShortDesc = "Set Management API configuration"
...
     var strPort string
     fmt.Scanln(&strPort)
     if len(strPort) == 0 {
-        fmt.Println("Port number is not specified, default value is used")
+        fmt.Println("Port number is not specified, default value 9154 is used")
```
Comment (Nirothipan) [inline cmd/cmd/init.go]: 9164

File: cmd/utils/constants.go [hunk 6]
```diff
@@ -84,10 +84,10 @@ const DefaultHttpRequestTimeout = 100000
 const HTTPProtocol = "http://"
 const HTTPSProtocol = "https://"
 const DefaultHost = "localhost"
-const DefaultPort = "9201"
+const DefaultPort = "9154"
```
Comment (Nirothipan) [inline cmd/utils/constants.go]: 9164

File: cmd/cmd/init.go [hunk 7]
```diff
@@ -32,35 +32,56 @@ const initCmdShortDesc = "Set Management API configuration"
...
+        fmt.Println("Host name is not specified, default value localhost is used")
         host = utils.DefaultHost
     }
-    fmt.Print("Enter Port number: ")
+    fmt.Print("Port number(default 9164): ")
     var strPort string
     fmt.Scanln(&strPort)
     if len(strPort) == 0 {
-        fmt.Println("Port number is not specified, default value is used")
+        fmt.Println("Port number is not specified, default value 9164 is used")
```
Comment (a5anka) [inline cmd/cmd/init.go]: Can we use the constant here for the port?

Comment (SanojPunchihewa) [reply cmd/cmd/init.go]: Fixed in 39767b9

**Pull request merged**